feature-libs/product/multi-dimensional/product-multi-dimensional.module.ts

```diff
@@ -0,0 +1,17 @@
+/*
+ * SPDX-FileCopyrightText: 2024 SAP Spartacus team <spartacus-team@sap.com>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { NgModule } from '@angular/core';
+import { ProductMultiDimensionalComponentsModule } from './components/product-multi-dimensional-components.module';
+import { ProductMultiDimensionalOccModule } from './occ/product-multi-dimensional-occ.module';
+
+@NgModule({
+  imports: [
+    ProductMultiDimensionalOccModule,
+    ProductMultiDimensionalComponentsModule,
+  ],
+})
+export class ProductMultiDimensionalModule {}
```

Comment:
[MEDIUM]
For consistency and correctness, lets rename all ocurrences of the word `Multidimensional` to a single word (instead of 2 words). It's a single word.
e.g.
`MultiDimensional` -> `Multidimensional`
`multi-dimensional` -> `multidimensional`
etc.

Note: this comment applies to many places in this PR

Reply:
In SAP wikis it's called Multi-Dimensional, should I proceed with this change?

Reply:
In general, I don't mind either option. Just let's be consistent.

My arguments for `multidimensional` as 1 word:
- Backend already returns us a property `product.multidimensional`
- it's correct according to EN dictionary https://dictionary.cambridge.org/dictionary/english/multidimensional

Reply:
I understand why you want to do it, but what I see is that a new term, "Multi-Dimensional," has been created specifically for this feature. It's referred to this way in every wiki. The way I see it, "multidimensional" and "Multi-Dimensional" are different terms, with the latter suggesting a particular feature.
e.g.
https://microlearning.opensap.com/media/Multi-Dimensional+Product+Variants+-+SAP+Commerce+Cloud/1_jmulme9b